readme.md

```diff
@@ -42,6 +42,45 @@ TBA
 
 ![db schema](https://github-production-user-asset-6210df.s3.amazonaws.com/19575839/261530664-6140d0e8-abf8-4fb8-80c9-ff9eb7adfcb6.png)
 
+```mermaid
+
+erDiagram
+
+    users {
+        id serial PK "not null"
+        phone varchar "not null, unique"
+        email varchar "not null"
+        first_name varchar "not null"
+        last_name varchar "not null"
+        password_hash varchar "not null"
+        password_salt varchar "not null"
+        group_id integer FK "not null"
+        created_at timestamp "not null"
+        updated_at timestamp "not null"
+    }
+
+    groups {
+        id serial PK "not null"
+        name varchar "not null"
+        key varchar "not null"
+        created_at timestamp "not null"
+        updated_at timestamp "not null"
+    }
+
+    business_details {
+        id serial PK "not null"
+        company_name varchar "not null, unique"
+        tax_number varchar "not null, unique"
+        owner_id integer FK "not null"
+        created_at timestamp "not null"
+        updated_at timestamp "not null"
+    }
+
+    users one or many -- one groups: "users(group_id) belongs to groups(id)"
```

Comment:
I believe something is wrong with relations. Now in your er it's described that 1 user can have multiple businesses, but our app will not work that way 🤔
We discussed two variants:
1 - one user can have one business account, and one business account belongs to 1 user, in that case we can place fk to business details table ( I believe we started with implementing this logic 🤔)
2 - one user can have one business account, and one business account can have multiple users, in this case fk business_id should be placed in user table

Reply:
Yes, but this what is implemented at the moment, we have set foreign keys between business_details and user tables but not relations, and now nothing stops several business_details have same userId.

Reply:
Ok, got it